Make.inc

```diff
@@ -958,7 +958,12 @@ else ifeq ($(OS), Darwin)
   RPATH_ESCAPED_ORIGIN := $(RPATH_ORIGIN)
   RPATH_LIB := -Wl,-rpath,'@loader_path/julia/' -Wl,-rpath,'@loader_path/'
 else
-  RPATH := -Wl,-rpath,'$$ORIGIN/$(build_libdir_rel)' -Wl,-rpath-link,$(build_shlibdir) -Wl,-z,origin
+ifeq ($(OS), FreeBSD)
+  RPATH := -Wl,-rpath,'$$ORIGIN/$(build_private_libdir_rel)'
+else
+  RPATH :=
+endif
+  RPATH += -Wl,-rpath,'$$ORIGIN/$(build_libdir_rel)' -Wl,-rpath-link,$(build_shlibdir) -Wl,-z,origin
```

Comment:
I think this can be more concise by initializing `RPATH` to the shared subset, then, if on `FreeBSD`, adding on the `FreeBSD`-specific part.  Ordering shouldn't matter.

Reply:
Okay. I wasn't sure if the order mattered, that's why I did it this way.